xml/cap_admin_upgrade.xml

```diff
@@ -131,51 +131,596 @@
+</screen>
+    </step>
+    <step>
+     <para>
+      Scale the entire <literal>scf</literal> deployment, including
+      the <literal>mysql</literal> role, back to the default
+      high availability mode.
+     </para>
+     <para>
+      Reuse the custom configuration file, called
+      <filename>ha-strict-true.yaml</filename> in this example,
+      created earlier. The file sets <literal>config.HA_strict</literal>
+      to <literal>true</literal>, which enforces all roles, including
+      <literal>mysql</literal>, are at the required minimum required instance
+      count for a default HA configuration. Note that it is assumed
+      <literal>config.HA=true</literal> is set in your existing
+      <filename>scf-config-values.yaml</filename> file.
+     </para>
+<screen>&prompt.user;helm upgrade susecf-scf suse/cf \
+--values scf-config-values.yaml \
+--values <replaceable>ha-strict-true.yaml</replaceable> \
+--version 2.19.1 \
+--set "secrets.UAA_CA_CERT=${CA_CERT}"
+     </screen>
+    </step>
+    <step>
+     <para>
+      If installed, upgrade Stratos.
+     </para>
+<screen>&prompt.user;helm upgrade --recreate-pods <replaceable>susecf-console</replaceable> suse/console \
+--values scf-config-values.yaml \
+--version 2.6.0
+</screen>
+    </step>
+   </procedure>
+  </sect2>
 
+  <sect2 xml:id="sec-cap-upgrade-custom-sizing">
+   <title>&ha; <literal>mysql</literal> through Custom Sizing Values</title>
+   <para>
+    This section describes the upgrade process for deployments where the
+    <literal>mysql</literal> roles of <literal>uaa</literal> and
+    <literal>scf</literal> are currently in high availability mode by
+    configuring custom sizing values (see <xref linkend="sec-cap-ha-custom"/>).
+   </para>
+
+   <procedure>
+    <step>
+     <para>
+      For the <literal>uaa</literal> deployment, scale the instance count of the
+      <literal>mysql</literal> role to 1 and remove the associated persistent
+      volume claim.
+     </para>
+     <substeps>
+      <step>
+       <para>
+        Scale the <literal>mysql</literal> instance count to 1.
+       </para>
+       <para>
+        Modify your existing custom sizing configuration file, called
+        <filename>uaa-sizing.yaml</filename> in this example, so that the
+        <literal>mysql</literal> role has a count of 1.
+       </para>
+<screen>sizing:
+  mysql:
+    count: 1
+</screen>
+       <para>
+        Perform the scaling.
+       </para>
+<screen>&prompt.user;helm upgrade susecf-uaa suse/uaa \
+--values scf-config-values.yaml \
+--values <replaceable>uaa-sizing.yaml</replaceable> \
+--version 2.18.0
+</screen>
+       <para>
+        Monitor progress using the <command>watch</command> command.
+       </para>
+       &watch-uaa;
+      </step>
+      <step>
+       <para>
+        Delete the persistent volume claim (PVC) for <literal>mysql-1</literal>
+	from the <literal>uaa</literal> namespace. <emphasis>Do not</emphasis>
```

Comment:
In this case the customer would need to delete all the orphaned PVCs for the rest of the instances, not just the one corresponding to mysql-1...